readthedocs/templates/projects/project_import.html

```diff
@@ -209,6 +209,26 @@ <h1>{% trans "Import a Repository" %}</h1>
         </div>
       {% endblock %}
 
+      {# accounts is a dictionary with the provider as key and a list of connected accounts of that provider #}
+      {% get_social_accounts user as accounts %}
+      {% if accounts %}
+        <h3>{% trans "Filter by Own Repositories" %}</h3>
+        {% for type, list in accounts.items %}
```

Comment:
By populating this by Django templates, we loose the ability to use observers and enable/disable the CSS class that make it gray. What should we do here?

Reply:
This should be driven by knockout in the same way that the organization listing is driven by knockout.

The best way to handle this might be to make the array of repositories an array of objects that specify if the object is a user repository or an organization repository. The organization filtering would be repurposed to know about both.

User repositories should come first in the array, so that listing is consistent.

Reply:
This makes sense to me. I didn't know how we want to implement this

* a new endpoint to retrieve all the user accounts?
* a `<script>` tag defining the JS object with django template tags?
* any other idea

Reply:
I created a new endpoint at https://github.com/rtfd/readthedocs.org/pull/3548/commits/a9d98a2856e378988a42a40e054b2ca7cbba3b3a